react-native-snap-carousel/README.md

```diff
@@ -0,0 +1,18 @@
+# Basic Reason Template
```

Comment:
actually the template we need to have is to have an `Installation` and then the building instructions